lib/sufia/export/converter.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia
+  module Export
+    # a base class to convert an ActiveFedora object that contains permissions and allow for pretty json.
+    #
+    class Converter
+      # overrides to_json to optionally allow for a pretty version of the json to be outputted
+      #
+      # @param [Boolean] pretty pass true to output formatted json using pretty_generate
+      def to_json(pretty = false)
+        return super unless pretty
+        JSON.pretty_generate(JSON.parse(to_json))
```

Comment:
@mjgiarlo & @hectorcorrea I tried super and that was no good (which is strange) calling to_json without the true then calls super.  Do not ask me why that is different from calling to_json, but the tests fail when I tried the change with the last PR.